lib/fs.js

```diff
@@ -196,10 +196,14 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this._atim_msec = atim_msec;
-  this._mtim_msec = mtim_msec;
-  this._ctim_msec = ctim_msec;
-  this._birthtim_msec = birthtim_msec;
+  this.atim_msec = atim_msec;
```

Comment:
Just so you know, this is/was named this way because it derives from the POSIX `stat` struct that has existed with these names for a long time; I think it’s okay to keep the underscore here.